applications/notifications.py

```diff
@@ -26,6 +26,14 @@ class NotificationType(TextChoices):
         "unmarked_winter_storage_application_created",
         _("Unmarked winter storage application created"),
     )
+    BERTH_APPLICATION_CONFIRMED = (
+        "berth_application_confirmed",
+        _("Berth application confirmed"),
+    )
+    WINTER_STORAGE_APPLICATION_CONFIRMED = (
+        "winter_storage_application_confirmed",
+        _("Winter storage application confirmed"),
+    )
```

Comment:
so, what are we gonna use these for?

Reply:
Not 100% sure when, but we have templates for `Application confirmation`, so just preventing that. HOWEVER, now that I think about it, they could (and maybe should) be left out of this PR.

Reply:
if we are not really using them anywhere, I would leave them out for now

Reply:
I am in general not sure what these are supposd to be for 🤔  I was thinking that `ORDER_APPROVED` already is `APPLICATION_HANDLED` notification as such

Reply:
Yeah, I'm not sure either, I was just preparing fot the templates that Tatu made :derp:

Reply:
voting for leaving it out for now

Reply:
if you mean these
![image](https://user-images.githubusercontent.com/24206160/93205008-c0fd4900-f75f-11ea-9e8b-7f3e3ab0da36.png)
then they are `BERTH_APPLICATION_CREATED`, `WINTER_STORAGE_APPLICATION_CREATED` and `UNMARKED_WINTER_STORAGE_APPLICATION_CREATED` effectively, but with new looks

Reply:
`application-confirmation-##.html` are all old templates updated to HDS. They might be a red herring in this discussion, check with Kalle

Reply:
AHA! Alright, then definitely no need for them 👍